Wplace API.md

```diff
@@ -113,6 +113,7 @@ Requires Authentication
 ### Leaderboard
 
 type = alliance, region/{countryID}, country, player, alliance
+
 timeframe = today, week, month or all-time
 
 countryID can be 0 to show all
```

Comment:
Please MD-Quote this, add "#### Query Params" heading above, and put it below Route.

Do the same for other API routes as well please.